### DISCUSSION TITLE
Implement a background video feature

### DISCUSSION THREAD

**Discussion by StrangeMcCarthy** [varabyte/kobweb#493] [Q&A]
Implement a background video feature

---

To ensure this feature gets triaged / prioritized quickly, please...

* describe what you'd like to be able to do that you currently can't
* feel free to include one (or more) solutions as a suggestion

**Comment by bitspittle** [comment]
Sorry, it's not clear what you're asking here. It also may not belong in Kobweb. This sounds like the sort of feature that should be in a different library or implemented per project.

**Comment by StrangeMcCarthy** [comment]
Ohh sorry I meant to implement a background video feature like that of a
background image. Like with HTML and CSS it is possible to put a video as
background of a container

**Reply by bitspittle** [reply]
Migrated to a discussion -- this is not a feature that Kobweb will support itself, because it's already supported by Compose HTML APIs.

Start here: https://www.w3schools.com/howto/howto_css_fullscreen_video.asp and then migrate those concepts over to Kobweb. 
This is how https://kobweb.varabyte.com includes video in its home page: https://github.com/varabyte/kobweb-site/blob/3a2c5eedec880f342b795b46406502729f8e1553/site/src/jsMain/kotlin/com/varabyte/kobweb/site/components/sections/home/CliSection.kt#L43
